src/multi.c

```diff
@@ -0,0 +1,146 @@
+/* Handle MULTI/EXEC transactions here.
```

Comment:
@tezc Are there changes here? It's very hard to review if that's the case. I suggest to keep one commit for changes and another for transfer of code.

Reply:
Yes, updated the PR. Please take a look at the first PR for the changes.

Reply:
Yes, updated the PR. Please take a look at the first PR for the changes.